Fixes NtOS chat bluescreen, muting

jlsnow301 opened the pull request:
<!-- Write **BELOW** The Headers and **ABOVE** The comments else it may not be viewable. -->
<!-- You can view Contributing.MD for a detailed description of the pull request process. -->

## About The Pull Request
- Could not actually mute other clients
- Turning on admin permanently deleted the user
- Screen sizing was bonk
**New screen sizing with muted users, admin mode on:**
![1](https://user-images.githubusercontent.com/42397676/146273002-720a2658-49be-4abb-a5b8-1cba97af3e4e.png)

<!-- Describe The Pull Request. Please be sure every change is documented or this can delay review and even discourage maintainers from merging your PR! -->

## Why It's Good For The Game
Fixes #62472 
- Chat client is a little easier to view, user panel is fully visible
- No more bluescreen
- (chat) admins can mute others
<!-- Please add a short description of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding. -->

## Changelog

<!-- If your PR modifies aspects of the game that can be concretely observed by players or admins you should add a changelog. If your change does NOT meet this description, remove this section. Be sure to properly mark your PRs to prevent unnecessary GBP loss. You can read up on GBP and it's effects on PRs in the tgstation guides for contributors. Please note that maintainers freely reserve the right to remove and add tags should they deem it appropriate. You can attempt to finagle the system all you want, but it's best to shoot for clear communication right off the bat. -->

:cl:
fix: Fixes a bluescreen when turning on admin mode in NtOS chat software.
fix: Literally 1984: NtOS chat admins can now properly mute others.
fix: Turning off NtOS admin mode now properly returns you to the channel.
fix: NtOS chat screen widened slightly to fix clipping issue on the participants panel.
/:cl:

<!-- Both :cl:'s are required for the changelog to work! You can put your name to the right of the first :cl: if you want to overwrite your GitHub username as author ingame. -->
<!-- You can use multiple of the same prefix (they're only used for the icon ingame) and delete the unneeded ones. Despite some of the tags, changelogs should generally represent how a player might be affected by the changes rather than a summary of the PR's contents. -->

jlsnow301 committed:
Fix is in

jlsnow301 committed:
Linter

jlsnow301 committed:
Bonk

stylemistake reviewed (approved):
I hate that your change is 90% formatting, but looks good.

jlsnow301 commented:
> I hate that your change is 90% formatting, but looks good.

Sorry ya 😑 If not for the linter it would be a 10 line PR

jlsnow301 committed:
check pass pls

jlsnow301 committed:
please pass

LemonInTheDark commented on code/modules/modular_computers/file_system/programs/ntnrc_client.dm:
```diff
@@ -91,8 +91,7 @@
 		if("PRG_toggleadmin")
 			if(netadmin_mode)
 				netadmin_mode = FALSE
-				if(channel)
-					channel.remove_client(src) // We shouldn't be in channel's user list, but just in case...
+				channel?.add_client(src)
 				return TRUE
```
why would we add the user when they de admin?

jlsnow301 replied:
so adminning removes the user, and this puts them back in the chat. otherwise, going admin mode permanently deletes the user from the chat. if you run around the station and admin the consoles, you can delete all the participants

jlsnow301 replied:
i believe this was an oversight but if adminning is meant to just permanently remove them i can undo this

Razharas commented:
Do I assume correctly that everything besides screen width and one column size all the rest of the template changes are discardable?

jlsnow301 commented:
> Do I assume correctly that everything besides screen width and one column size all the rest of the template changes are discardable?

yes outside of prettier hitting it, it's relatively small. the fix was in a nullcheck I added to channel users,
```disabled={this_client?.muted}``` and ```tooltip={(!this_client?.muted && 'Ping') || 'You are muted!'}```

+ added window width and smaller grow size for chat panel

jlsnow301 committed:
Clarity

jlsnow301 commented:
@Razharas I have further clarified this by undoing the changes made by my formatter
